apps/full-site-editing/full-site-editing-plugin/donations/fetch-default-products.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable wpcalypso/import-docblock */
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const fetchDefaultProducts = async ( currency ) => {
+	try {
+		const result = await apiFetch( {
+			path: `/wpcom/v2/memberships/products?type=donation&currency=${ currency }`,
```

Comment:
Optional: just noting that parameters can be also set in a `data` object:

```
{
  path: '/wpcom/v2/memberships/products',
  method: 'POST',
  data: {
    type: 'donation',
    currency,
  }
}
```

Reply:
Oh, ha! I tried that, and saw that it didn't include it in the URL, so I assumed the calls would fail – but you're right, they succeed just fine 👍 0b939946a5fffd39a004e0936261bb46a10dbde7